policy/modules/system/authlogin.fc

```diff
@@ -24,34 +22,20 @@ HOME_DIR/\.google_authenticator~		gen_context(system_u:object_r:auth_home_t,s0)
 /etc/ptmptmp		--	gen_context(system_u:object_r:passwd_file_t,s0)
 /etc/group[-\+]?	--	gen_context(system_u:object_r:passwd_file_t,s0)
 
-/sbin/pam_console_apply	 --	gen_context(system_u:object_r:pam_console_exec_t,s0)
-/sbin/pam_timestamp_check --	gen_context(system_u:object_r:pam_timestamp_exec_t,s0)
-/sbin/tcb_convert	--	gen_context(system_u:object_r:updpwd_exec_t,s0)
-/sbin/tcb_unconvert	--	gen_context(system_u:object_r:updpwd_exec_t,s0)
-/sbin/unix_chkpwd	--	gen_context(system_u:object_r:chkpwd_exec_t,s0)
-/sbin/unix_update	--	gen_context(system_u:object_r:updpwd_exec_t,s0)
-/sbin/unix_verify	--	gen_context(system_u:object_r:chkpwd_exec_t,s0)
-ifdef(`distro_suse', `
-/sbin/unix2_chkpwd	--	gen_context(system_u:object_r:chkpwd_exec_t,s0)
```

Comment:
Removing this line (or three) was intentional?

Reply:
Hmm, I thought that the file is listed below, but it's `unix_chkpwd`, not `unix2_chkpwd`. I'll add it back.